scripts/monitor/persistEvents.js

```diff
@@ -53,6 +55,13 @@ export async function processEvent(event) {
       }
       log.info(`[${name}] Creating publication ${contract_id} for ${parcelId}`)
 
+      await Publication.delete({
+        x,
+        y,
+        owner: seller.toLowerCase(),
```

Comment:
Do we need this? It may be better to skip `owner` altogether...

Reply:
Do you mean nos filtering by owner?

Reply:
Right

Reply:
I added it to avoid being to eager deleting publications _if_ we're replaying the monitor from the beginning. While the data is not lost, if everything is working correctly publications from other users should be cancelled or sold, no?

Reply:
@eordano do you see a possible problem by constraining to owner?